templates/profiles/change_password.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load markdownify %}
+{% load courses_extra %}
+{% load bootstrap_tags %}
+
+{% block content %}
+<form method="post">
+    {% csrf_token %}
+    <div class="py-5 container bg-light">
+            <table style="margin-right: auto; margin-left: auto">
```

Comment:
No need for table and inline style (this can be achieved with bootstrap if necessary)...